fleet_notifications/state_checker.py

```diff
@@ -3,10 +3,15 @@
 from fleet_management_http_client_python import ApiClient, CarApi, OrderApi, OrderStateApi, OrderStatus
 from fleet_notifications.notifications_client import NotificationClient
 from fleet_notifications.database.database_controller import get_orders, update_order, delete_order
```

Comment:
When I jump in the middle of the code and find method 'delete_order', I often have to look from where the function is imported to know, where the order is deleted (on management api or notifications api). 

I suggest to change the import to show, to which api the method relates, more explicitly.
```suggestion
import fleet_notifications.database.database_controller as notifications_db
```